.github/workflows/Main.yml

```diff
@@ -36,7 +36,7 @@ jobs:
       run: sudo apt-get update -y -qq && sudo apt-get install -y -qq ninja-build ccache
 
     - name: Build
-      run:  make debug
+      run:  BUILD_SUBSTRAIT_EXTENSION=1 BUILD_TPCH=1 make debug
 
```

Comment:
Can we add the extension to the installable extensions tests & deployments (`win-extensions-64`, `linux-extensions-64`, `xcode-extensions`)?